### PR TITLE
replacing datetime with arrow

### PR DESCRIPTION
**Pull request by achasmita**
fixed issue #73

### CODE REVIEW

**Comment by achasmita** [comment]
Data is same before and after changes for trajectory and trip table:
Screen Shot 2023-10-18 at 11 06 22 PM
Screen Shot 2023-10-18 at 11 06 39 PM

**Comment by achasmita** [comment]
Since there is no data in recent 7 days, the table appears blank
Screen Shot 2023-10-20 at 9 13 42 AM
We can select date from filter and load data accordingly:
Screen Shot 2023-10-20 at 9 15 12 AM

**Review by shankari — CHANGES_REQUESTED** [review]
This has the hallmarks of a hastily slapped together set of changes.
- What is the rationale for some of some of the changes?
- Why are we filtering the trajectory table when working on #73? Is this also a fix for #51?
   - In which case, why are we filtering only the trajectory table?
- Where is the testing done?

Please make sure that PRs that are submitted are clean and actually ready for review.

File: utils/db_utils.py [hunk 1]
```diff
@@ -108,9 +108,9 @@ def query_demographics():
     entries = ts.find_entries(["manual/demographic_survey"])
     df = pd.json_normalize(list(entries))
     if not df.empty:
-        for col in constants.BINARY_DEMOGRAPHICS_COLS:
-            if col in df.columns:
-                df[col] = df[col].apply(str) 
+        for col in df.columns:
+            if df[col].dtype == 'object':
+                df[col] = df[col].apply(str)
```
[the remark this answers is not in the document]
Comment (achasmita) [reply utils/db_utils.py]: I have reverted this change as  this change is already included in another commit.

**Pull request closed without merging**